code/_onclick/observer_onclick.dm

```diff
@@ -39,12 +39,16 @@
 			M = get_mob_in_atom_with_warning(A)
 			if(M)
 				client.holder.show_player_panel(M)
+			CtrlClickOn(A)
```

Comment:
For admins, I suggest doing if they CANT find a mob
```suggestion
				client.holder.show_player_panel(M)
			else
				CtrlClickOn(A)
```

Reply:
Makes you unable to scan on the same tile as a mob. I could just check if you clicked on a mob directly but that would make it a bit harder to open the panel so maybe not.

Reply:
I think that its fine in adminmode you cant always scan pipes if someones standing over them